login_test.rb

```diff
@@ -3,53 +3,71 @@
 require 'test/unit'
 include Test::Unit::Assertions
 
-puts "Enter Username: "
-userID = gets
-puts "Enter Password: "
-pass = gets
-
-driver = Selenium::WebDriver.for :edge
-wait = Selenium::WebDriver::Wait.new(:timeout => 10)
-# userID = "ID"
-# pass = "PASS"
-cartItem = "ITEM"
-
-
-driver.navigate.to "https://www.tokopedia.com/"
-
-findLogin = wait.until {driver.find_element(xpath: "//button[@data-testid= 'btnHeaderLogin']")}
-findLogin.click
-
-enterID = wait.until {driver.find_element(xpath: "//input[@data-testid= 'email-phone-input']")}
-enterID.send_keys +userID
-
-proceedToPass = wait.until {driver.find_element(xpath: "//button[@data-testid= 'email-phone-submit']")}
-proceedToPass.click
-
-enterPass = wait.until {driver.find_element(xpath: "//input[@autocomplete= 'current-password']")}
-enterPass.send_keys +pass
-
-login = wait.until {driver.find_element(xpath: "//span[@aria-label= 'login-button']")}
-login.submit
-
-verify = wait.until {driver.find_element(xpath: "//img[@src = 'https://ecs7.tokopedia.net/otp/cotp/ICON_EMAIL_NEW.png']")}
-verify.click
-
-puts "Enter verification code: "
-verificationCode = gets
-
-otp = wait.until {driver.find_element(xpath: "//input[@autocomplete = 'one-time-code']")}
-otp.send_keys +verificationCode	
-
-inputItemName = wait.until {driver.find_element(xpath: "//input[@data-unify= 'Search']")}
-inputItemName.send_keys +cartItem
-
-searchItem = wait.until {driver.find_element(xpath: "//button[@aria-label= 'Tombol pencarian']")}
-searchItem.submit
-
-#findCorrectItem = wait.until {driver.find_element(xpath: "//a[contains(@title= 'IPhone 13 Pro Max')]")}
-#findCorrectItem.click
-
-#addToCart = wait.until {driver.find_element(xpath: "//button[@data-testid = 'pdpBtnNormalPrimary']")}
-#addToCart.click
-
+driver = Selenium::WebDriver.for :chrome
+Login_url = "https://accounts.bukalapak.com/login?"
+User = "user"
+Pass = "pass"
+Item_to_add = "item"
+Login_button = "//a[@class= 'sigil-header__nav te-header-login']"
+Email_field = "//div[@dclass= 'bl-text-field__boxed']"
+To_password_field = "//button[@id= 'submit_button']"
+Password_field = "//div[@class= 'bl-text-field__boxed']"
+Login = "//div[@id= 'btn-login']"
+Search_field = "//input[@name= 'search[keywords]']"
+Search_button = "//button[@title= 'Cari']"
+Correct_item = "//a[contains(@title= '#{Item_to_add}')]"

```

Comment:
daripada gini dibikin jadi method aja

```suggestion
def correct_item(item_to_add)
      "//a[contains(@title= '#{Item_to_add}')]"
end
```